jekyll/_cci2/gitlab-integration.adoc

```diff
@@ -126,18 +64,28 @@ A project can have one or more **triggers**, which are events from a source of c
 
 The following settings are found by clicking the **Project Settings** button within your project. At this time, both configurations and triggers are limited to GitLab. To read more about other settings you can enable for your projects, refer to the <<settings#,Settings>> document.
 
-[#project-settings-active-development]
-=== Project settings in active development
+[#people]
+=== People
+
+Project roles give control over which users have access to which projects within an organization. This enables teams to have limited access to only their projects, while managers and others can have broader organizational access. The access options are: 
+
+* Admin: Read and write access to the project and all settings and ability to manage other users' access.
+* Contributor: Read and write access to the project and some settings.
+* Viewer: Read only access to the project and some settings.
+
+For a complete list of permissions, see the <<#permissions-matrix>>.
```

Comment:
I think we'll also need to add a new section to this table for "Project Roles" to complement the existing "Organization Roles". I'm not certain how best to modify the table format... but in case this is a quick shortcut for you to make the change, https://github.com/circleci/permissions-service/blob/16b3fea866260575651ec154032d2afe4f0adf3b/service/src/circleci/permissions/roles.clj#L186 has the full list of actions for these roles. @nathanfishcircleci , do you think you could assist w/ clarifications as needed or rope our team in if not?